drivers/eeprom/include/eeprom_params.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2019 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup   drivers_eeprom
+ * @{
+ *
+ * @file
+ * @brief     Initialization parameters for the EEPROM drivers
+ *
+ * @author    Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+ */
+
+#ifndef EEPROM_PARAMS_H
+#define EEPROM_PARAMS_H
+
+#include "board.h"
+#include "eeprom_driver.h"
+
+#ifdef MODULE_AT24CXXX
+#include "at24cxxx_params.h"
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifdef MODULE_PERIPH_EEPROM
+extern const eeprom_driver_t eeprom_driver_cpu;
+#endif
+
+#ifdef MODULE_AT24CXXX
+extern const eeprom_driver_t eeprom_driver_at24cxxx;
+static at24cxxx_t _at24cxxx_handle;
+#endif
+
+/**
+ * @brief   The EEPROM drivers
+ */
+static const eeprom_dev_t eeprom_devs[] = {
+#ifdef MODULE_PERIPH_EEPROM
+    {
+        .driver = &eeprom_driver_cpu,
+        .params = NULL,
+        .handle = NULL,
+    },
+#endif
+#ifdef MODULE_AT24CXXX
+    {
```

Comment:
How about this (list of) struct is taken from a macro defined in AT24CXXX params header ? This would allow to configure and use multiple external EEPROM of the same kind. The default driver params could also be overloaded the usual way.

Reply:
I like the idea. I implemented it a bit different to allow only changing only the part of the array related to a single driver. E.g. if I had 2 AT25*, one AT24C* and one internal EEPROM device, this would allow to overwrite the AT25* related part of the array.